environment.yml

```diff
@@ -38,3 +38,8 @@ dependencies:
   - mpmath
   - gmpy2
   - threadpoolctl
+  # For CLI
+  - rich-click
+  - click
+  - pip:
```

Comment:
We an already remove this I think since the release on conda is in progress.

Reply:
Updated, though for `arm64` it failed with `PackagesNotFoundError:`. It's not yet available under `conda-forge/osx-arm64` or `conda-forge/noarch
`

Reply:
Opened an issue for that at https://github.com/conda-forge/doit-feedstock/issues/16

Reply:
Looks like we won't get that arm64 package, so let's revert back to `pip` until there's a new `doit` release which is `noarch`.